pipelines/backup/mongodb-backup/config.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?><flow-definition plugin="workflow-job@2.40">
```

Comment:
In this file has a `hosts` job parameter and set the default value to the host you want to take the backup from.